src/solver/conv_winoRxS_fused.cpp

```diff
@@ -45,8 +45,107 @@
 
 MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F2X3_G1)
 
+#define IS3X2 (Winodata == 3 && Winofilter == 2)
+
+static inline size_t Ceil(const size_t v, const size_t m)
+{
+    assert(m > 0);
+    return (v + m - 1) / m;
+}
+
 namespace miopen {
 namespace solver {
+
+namespace {
+
+// Winograd v30 is functionally supported on Vega10/Vega20 ASICs, but performance regression is
+// expected to be ~25%. Use Winograd v21 instead.
+// Details: https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1927#issuecomment-1412741130
+template <int Winodata, int Winofilter>
+inline bool IsWinogradV30Supported(const std::string& asic, const ProblemDescription& problem)
```

Comment:
Both V30 and V21 are supported on vega. Suggest to use `IsWinogradV21Preffered()` instead (and change logic accordingly).